### PR TITLE
Debugging Project

### PR DESCRIPTION
**Pull request by kaymanq**
1.The system of relationships in the database has been redesigned.
2.fixed all errors in project tree
3.Assigned myself to PR
4. Edit title in PR
5. Waiting some new quest, Master

### CODE REVIEW

File: models/relationships.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const User = sequelize.define('user', {
+    id: {type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true, unique:true},
+    email: {type: DataTypes.STRING, unique: true},
+    password: {type:DataTypes.STRING},
+    username: {type:DataTypes.STRING}
+})
+const Place = sequelize.define('place', {
+    id: {type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true, unique:true},
+    name: {type: DataTypes.STRING},
+    discription: {type:DataTypes.STRING},
```
Comment (return-none) [inline models/relationships.js]: should be `TEXT` data type

**Review by return-none — CHANGES_REQUESTED** [review]
1. please, provide a meaningful PR title
2. please, update PR description to reflect changes introduced
3. assign yourself as assignee of PR
4. each model should be defined in own file, relationships can be defined in one file, use `index.js` name in `models` dir

File: models/relationships.js (hunk 1 above)
Comment (kaymanq) [reply models/relationships.js]: fixed

File: server.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 require('dotenv').config()
 const express = require ('express');
 const sequelize = require('./db')
-const PORT = process.env.PORT || 5000;
+const models = require('./models/relationships')
```
Comment (return-none) [inline server.js]: `relationships` is no longer needed, everything is defined in index file now

File: server.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
+const PORT = process.env.PORT || 5002;
```
Comment (return-none) [inline server.js]: why fallback for `PORT` has changed?

**Review by return-none — CHANGES_REQUESTED** [review]
1. Title is still wrong, try something like `Base Models`
2. Cannot see the PR description updated
3. You're still not assigned to PR

File: server.js (hunk 2 above)
Comment (kaymanq) [reply server.js]: fixed

File: server.js (hunk 3 above)
Comment (kaymanq) [reply server.js]: fixed

**Comment by kaymanq** [comment]
1.fixed all errors in project tree
2.Assigned myself to PR
3. Edit title in PR
4. Waiting some new quest, Master

**Pull request closed without merging**